docs/api/crash-reporter.md

```diff
@@ -59,7 +59,7 @@ The `crashReporter` module has the following methods:
   * `rateLimit` Boolean (optional) _macOS_ _Windows_ - If true, limit the
     number of crashes uploaded to 1/hour. Default is `false`.
   * `compress` Boolean (optional) - If true, crash reports will be compressed
-    and uploaded with `Content-Encoding: gzip`. Default is `false`.
+    and uploaded with `Content-Encoding: gzip`. Default is `true`.
```

Comment:
The default here is still `false` right?

Reply:
oops, yep.
```suggestion
    and uploaded with `Content-Encoding: gzip`. Default is `false`.
```